app/views/hosts/_insights_tab.html.erb

```diff
@@ -0,0 +1,15 @@
+<% content_for(:javascripts) do %>
+  <%= webpacked_plugins_js_for :'foreman_rh_cloud' %>
+<% end %>
+
+<% content_for(:stylesheets) do %>
+    <%= webpacked_plugins_css_for :'foreman_rh_cloud' %>
+<% end %>
+
+<div id="host_details_insights_react_container"></div>
+<%= mount_react_component(
+  'InsightsHostDetailsTab',
+  '#host_details_insights_react_container',
+  { hostname: host.name }.to_json,
```

Comment:
You can pass `host.id` here, and then the route will be a bit more simple.

Reply:
how would the route be simpler? something like:
`get 'hits/:id', to: 'hits#show', constraints: { id: %r{[^\/]+} }`
instead of:
`get 'hits/:host_name', to: 'hits#show', constraints: { host_name: %r{[^\/]+} }`